### ISSUE TITLE
Take Photo with App in Offline Mode (no Internetconnection possible)

### ISSUE THREAD

**Issue by FiC5020** [nextcloud/android#7380]
### Steps to reproduce
1.  Open then Nextcloud client on the Tablet. Sync some folder and files for Offline Usage.
2.  Cut the Internet Connection of the Tablet
3.  Go in the Nextcloud to the folder which is sync for Offline Usage
4.  Take some Photos with the Function "Upload Form camera"
5.  The Photos now in the Nextcloud Section Uploads under Failed/Pending with reason Connection error, which ist correct.
6.  Now I back in office or have some Internetconnection.
7.  The first picture a took, uploaded to the server, the others not. under Failed Pending (Local file not found)

### Expected behaviour
The Client should collect the photos which i made while i'm offline in the folder where i made the photo.
When I go online, should ask, whether I want to synchronize.

### Actual behaviour
see above. I make photo wenn offline, dont See this pictures, an when Internet is back, the last will be synchronized.
Same behaviour, wenn internet drops unexpected for some time (some minutes or secondes). Photo in this periode will not be 
synchronized. Only the last one, The others or im upload log as "Local file not found".
https://cl.spirk.at/s/3kQH35HJ8razt5s

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version: 10

Device model: Samsung Galaxy Tab Active Pro (SM-T540)

Stock or customized system: Stock

Nextcloud app version: 3.13.1

Nextcloud server version: 18.01

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky atm this is by design, we simply save a file with a fixed name (to find it for upload...) so without an internet connection where you do this one photo at a time we do have a technical design issue here. 
Using a sequence might solve it but would take some effort to do so :/ (same applies for the document scan PR as I mentioned in the PR review).

**Comment by tobiasKaminsky** [comment]
Duplicate of #4609

**Comment by AndyScherzinger** [comment]
I created a PR that should fix the issue: #7411 - please test if possible and provide feedback if it solves the issue (testing showed it works for me) --> https://github.com/nextcloud/android/pull/7411#issuecomment-731435608
